pkg/apis/apis/v1alpha1/permissionclaims/labels.go

```diff
@@ -39,6 +39,20 @@ func ToLabelKeyAndValue(exportClusterName logicalcluster.Name, exportName string
 	return apisv1alpha1.APIExportPermissionClaimLabelPrefix + exportHash, claimHash, nil
 }
 
+// ToReflexiveAPIBindingLabelKeyAndValue returns label key and value that is set (as fallback for filtering)
```

Comment:
why is this part of the api pkg?

Reply:
because it is an API, a contract between internal components.